client/reader/search-stream/index.jsx

```diff
@@ -244,7 +268,8 @@ const SearchStream = React.createClass( {
 				className="search-stream" >
 				{ this.props.showBack && <HeaderBack /> }
 				<DocumentHead title={ this.props.translate( '%s ‹ Reader', { args: this.state.title || this.props.translate( 'Search' ) } ) } />
-				<div className="search-stream__fixed-area">
+				<div ref={ this.handleStreamMounted } />
```

Comment:
Is it possible to add this `ref` on `<Stream>` itself so we're not adding an extraneous div?